yarn.lock

```diff
@@ -0,0 +1,96 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+"@types/node@*":
```

Comment:
Do you need a yarn.lock file in a library?

Reply:
I think we should have it:
https://classic.yarnpkg.com/en/docs/yarn-lock#toc-check-into-source-control

> During install, Yarn will only use the top-level yarn.lock file and will ignore any yarn.lock files that exist within dependencies.

> Framework and library authors should also check yarn.lock into source control. Don’t worry about publishing the yarn.lock file as it won’t have any effect on users of the library.

Do you see any reason why not?

Reply:
The only one I can think of is that it not having it would make CI always use the latest accepted versions instead of the pinned versions. However, CI won't run on this very often so it likely doesn't matter.

Reply:
On some of my projects I put CI on a cron just to make sure nothing breaks.